algebird-spark/src/main/scala/com/twitter/algebird/spark/package.scala

```diff
@@ -0,0 +1,46 @@
+package com.twitter.algebird
+
+import org.apache.spark.rdd.{ RDD, PairRDDFunctions }
+
+import scala.reflect.ClassTag
+import com.twitter.algebird._
+
+/**
+ * To use this, you probably want:
+ * import com.twitter.algebird.spark._
+ */
+package object spark {
+  /**
+   * This adds methods to Spark RDDs to use Algebird
+   */
+  implicit class AlgebirdRDD[T](val rdd: RDD[T]) extends AnyVal {
+    /**
+     * Apply an Aggregator to return a single value for the whole RDD
+     */
+    def aggregatorOnAll[B: ClassTag, C](agg: Aggregator[T, B, C]): C =
+      agg.present(rdd.map(agg.prepare).reduce(agg.reduce))
+
+    def aggregatorByKey[K: ClassTag, A: ClassTag, B: ClassTag, C](agg: Aggregator[A, B, C])(implicit ev: T <:< (K, A), ordK: Ordering[K] = null): RDD[(K, C)] =
```

Comment:
- Where/how is `ev` / `T` used?  Is it needed by Spark or Algebird behind the scenes?

- Any particular reason to give a default of `null` to `ordK` and/or to use an implicit parameter instead of `K: ClassTag: Ordering`?

    ```scala
    implicit ordK: Ordering[K] = null
    ```